libpdf/core.py

```diff
@@ -409,6 +416,12 @@ def handle_parse_result(self, ctx, opts, args):
     show_default=True,
     help='Output directory for extracted figures; if it does not exist, it will be created',
 )
+@click.option(
+    '--no-annotations',
+    is_flag=True,
+    show_default=True,
+    help='Do not extract annotations from catalog. The links will not be resolved.',
```

Comment:
All PDF-internal links will not be resolved. Chapter detection however will still work.